### DISCUSSION TITLE
Vertical resize on floating window does not work

### DISCUSSION THREAD

**Discussion by jmiv-eng** [i3/i3#4789] [Q&A] [answered]
For some reason, floating windows are not able to be resized in the vertical direction. I have tried using the $mod+r and arrow keys, as I have set up those keybindings in my config file. The screencast below shows the suspect behavior (tested on the qt5ct program but also reproducible for most other floating windows). In the screencast, I try to use the $mod+r and arrow keys to resize the window (which doesn't work vertically but does horizontally), then use the $mod+rightclick method which again shows only horizontal mobility. Finally I have executed the xprop comand on the floating window if it is of any use. Any advice is appreciated. 

[Issue](https://user-images.githubusercontent.com/39566785/149386727-168d9935-ebef-4bb3-a27f-71dcb730af7f.mp4)

[I3 Config](https://github.com/jmichael16/dotfiles/blob/main/i3/.config/i3/config)

xprop output (if applicable):
```
I3_FLOATING_WINDOW(CARDINAL) = 1
_NET_WM_USER_TIME(CARDINAL) = 7777487
_NET_WM_DESKTOP(CARDINAL) = 1
_NET_WM_STATE(ATOM) =
WM_STATE(WM_STATE):
		window state: Normal
		icon window: 0x0
_NET_WM_ICON(CARDINAL) = 	Icon (32 x 32):
_NET_WM_ICON_NAME(UTF8_STRING) =
XdndAware(ATOM) = BITMAP
WM_NAME(STRING) = "Qt5 Configuration Tool"
_NET_WM_NAME(UTF8_STRING) = "Qt5 Configuration Tool"
_MOTIF_WM_HINTS(_MOTIF_WM_HINTS) = 0x3, 0x3e, 0x7e, 0x0, 0x0
_NET_WM_WINDOW_TYPE(ATOM) = _NET_WM_WINDOW_TYPE_NORMAL
_XEMBED_INFO(_XEMBED_INFO) = 0x0, 0x1
WM_CLIENT_LEADER(WINDOW): window id # 0x800008
WM_HINTS(WM_HINTS):
		Client accepts input or input focus: True
		window id # of group leader: 0x800008
_NET_WM_PID(CARDINAL) = 211801
_NET_WM_SYNC_REQUEST_COUNTER(CARDINAL) = 8388615
WM_CLASS(STRING) = "qt5ct", "qt5ct"
WM_PROTOCOLS(ATOM): protocols  WM_DELETE_WINDOW, WM_TAKE_FOCUS, _NET_WM_PING, _NET_WM_SYNC_REQUEST
WM_NORMAL_HINTS(WM_SIZE_HINTS):
		user specified location: 2353, 486
		user specified size: 957 by 712
		program specified minimum size: 594 by 712
		window gravity: Static
```

**Comment by Airblader** [comment, marked as answer]
Your config defines a maximum height for all floating windows:

```
floating_maximum_size -1 x 500
```

**Reply by jmiv-eng** [reply]
Thank you! Don't know how I missed this in my config file.
